program/tests/context.rs

```diff
@@ -371,6 +371,19 @@ impl Context {
             .expect("Failed to send memo transaction.")
     }
 
+    /// Warp to the given epoch after the first normal slot.
+    ///
+    /// Note, the epoch number here is not the same as the epoch number of the
+    /// clock sysvar; we start counting from the first "normal" slot.
+    pub fn warp_to_normal_epoch(&mut self, epoch: u64) {
```

Comment:
We can't warp to an epoch in the past, right? Maybe call it `warp_to_future_epoch`?

Reply:
Great function 👍, thanks!

Reply:
I’m not sure if it is possible, but it makes no sense. How about `advance_to_normal_epoch`? I put `normal` in there because it starts counting at `genesis_config.first_normal_slot`, so the first “normal” epoch is not actually the first epoch.

Reply:
Yeah, that's fine 👍